source/tutorial/create-system-dsn.txt

```diff
@@ -12,41 +12,161 @@ Create a System DSN
+           `MongoDB Enterprise <http://www.mongodb.com/products/mongodb-enterprise?jmp=docs>`__. 
+         - Configure your :binary:`~bin.mongod` and 
+           :binary:`~bin.mongos` instances to use 
+           :manual:`Kerberos (GSSAPI) authentication </core/kerberos>`.
+         - :manual:`Create a MongoDB database </core/databases-and-collections>` to which you want to connect 
+           using |bi-short|.
+         - `Create a keytab file`_ on the |kdc| that has the needed 
+           Kerberos :ref:`service principals <kerberos-service-principal>` 
+           for ``mongosql`` and ``mongosql2``.
+
+           .. example::
+
+              .. code-block:: sh
+              
+                 mongosql/<fullHostnameOfBIConnector>@<KERBEROS_REALM>
+ 
+         - Copy the ``keytab`` file you created for your MongoDB 
+           deployment to the same host that serves the |bi-short|. 
+         - If your keytab does not use the default name 
+           (``krb5.keytab``), you must set the 
+           `KRB5_KTNAME <https://web.mit.edu/kerberos/krb5-latest/doc/basic/keytab_def.html>`_ system variable.
+
+           .. note::
+
+              The Kerberos implementation on macOS, `Heimdal`_, does
+              not support the ``default_keytab_name`` configuration 
+              setting, like MIT Kerberos does. You must set the 
+              ``KRB5_KTNAME`` system variable. 
+
+         - If your Kerberos configuration file does not use the 
+           default name (``krb5.conf``), you must set the
+           `KRB5_CONFIG <https://web.mit.edu/kerberos/krb5-devel/doc/admin/conf_files/krb5_conf.html>`_ 
+           system variable. (Heimdal also supports 
+           `this system variable <https://www.h5l.org/manual/HEAD/info/heimdal/Configuration-file.html#Configuration-file>`_.)
+
+         - If you are using a ``mongosqld`` configuration file, you
+           must add the following settings.
+
+           **MongoDB BI Connector Authentication**
+
+           - :setting:`security.enabled` ``: true``
+           - :setting:`security.defaultMechanism` ``: "GSSAPI"``
+           - :setting:`security.defaultSource` ``: "$external"``
+           - :setting:`security.gssapi.hostname` 
+             ``: <host running mongosqld>``
+           - :setting:`security.gssapi.serviceName` 
+             ``: <name of mongosqld service>``
+             (:setting:`security.gssapi.hostname` should match the value in :setting:`serviceName`.)
+
+           .. example::
+
+              .. code-block:: yaml
+   
+                 security
+                   enabled : true
+                   defaultMechanism : "GSSAPI"
+                   defaultSource : "$external"           
+                   gssapi
+                     hostname : <host running mongosqld>
+                     serviceName : <name of mongosqld service>
+
+           **MongoDB Authentication**
+
+           - :setting:`mongodb.net.uri` 
```

Comment:
I think it's worth exploring my suggestions above, but this does look a bit easier on the eyes here for some reason...

Reply:
Spacing from the header. This is a CSS issue, for sure.